### ISSUE TITLE
D2 swarm

### ISSUE THREAD

**Issue by leandro-lucarella-sociomantic** [sociomantic-tsunami/swarm#290]
Just for completeness of the D2 migration process.

**Comment by leandro-lucarella-sociomantic** [comment]
Already in stage 5 (tags are auto converted to D2).

**Comment by mathias-lang-sociomantic** [comment]
Wait, is stage 5 different for libraries ?

**Comment by leandro-lucarella-sociomantic** [comment]
Yes. We can't drop D1 support for libraries until everything is D2-only. And we don't need to auto-convert tags for applications, so the requirements for apps and libraries are quite different. Also, once all apps are D2-only, there isn't much else to do with libraries, dropping D1 support is trivial.
